### ISSUE TITLE
(Bug report) `. For help visit wiki. `

### ISSUE THREAD

**Issue by Myzel394** [zadam/trilium#2546]
### Trilium Version

0.49.4

### What operating system are you using?

Ubuntu

### What is your setup?

Server access only

### Operating System Version

Linux myzel394-computer 5.15.0-2-amd64 #1 SMP Debian 5.15.5-2 (2021-12-18) x86_64 GNU/Linux

### Description

Trying out the new sharing functionality. However, I only get to see this message:

`
![image](https://user-images.githubusercontent.com/50424412/149327459-7e940087-73a7-4302-a74d-8e9a0bb3e6f4.png)

There are no errors in the browser console and all requests succeed with `200`.

**Comment by zadam** [comment]
Hi, is it reproducible?

Are you using the released 0.49.4 or perhaps master branch?

**Comment by Myzel394** [comment]
I'm using `0.49.4` (from DockerHub)

**Comment by zadam** [comment]
Hmm, unfortunately I can't reproduce that on a demo document on the docker image.

There were some bugs regarding search which could influence this, but there would be errors in the console. But just to rule this out, does this still happen if you restart the application (also backend) and look into this immediately (without doing anything else)?

**Comment by Myzel394** [comment]
Unfortunately that didn't work :/

No errors in backend either, however, this line seems a bit strange, doesn't it?
![image](https://user-images.githubusercontent.com/50424412/149407613-14d7ac6b-044c-44c7-8aca-f5ecc378f85d.png)



btw, I get a lot of these warnings in the browser, but I don't think it is related to this issue:

![image](https://user-images.githubusercontent.com/50424412/149407703-6e4a2a41-6fb9-4bed-b6f9-f2598a856892.png)

**Comment by zadam** [comment]
The BBBB is just a left-over debug message.

The clone-to-note is fine, except that it's a bit slow.

If you'd be willing to create an [anonymized database](https://github.com/zadam/trilium/wiki/Troubleshooting#anonymized-database) and send it to me at zadam.apps@gmail.com, I might be able to reproduce the issue ...

**Comment by Myzel394** [comment]
Okay I sent you a link to the database, please check your mails

**Comment by zadam** [comment]
Thank you. Unfortunately sharing seems to work fine on the anonymized database.

I'm at a loss what could be the issue.

I noticed that there is some custom widget (of course anonymized) - it's a long shot, but could it interfere here? Might be worth disabling it.

**Comment by Myzel394** [comment]
Okay I found the problem:

The `<span />` element IS actually present, however it has the style `display: none !important` in Firefox only. So I can see the span in Chrome and access the shared link both in Chrome and Firefox. I also think the problem is not in Firefox itself but in a extension that's inserting stylesheets (I think it's uBlock Origin, because it's a selector with like 1000 elements).

**EDIT**

Okay I found the problem! So apparently my adblocker thinks that all elements with the class "share-text" are for **sharing to social media**. Maybe we could change the class name?

Here's (part of) the stylesheet:

![image](https://user-images.githubusercontent.com/50424412/149572390-ebc8a9fa-6729-4652-b96e-922856da991d.png)
